lib/logstash/outputs/sns.rb

```diff
@@ -24,6 +24,12 @@
 #  with the @message truncated so that the length of the JSON fits in
 #  `MAX_MESSAGE_SIZE_IN_BYTES`.
 #
+# ## Upgrading to 2.0.0
+#
+# This plugin used to have a `format` option for controlling the encoding of messages prior to being sent to SNS.
+  # This plugin now uses the logstash standard https://www.elastic.co/guide/en/logstash/current/configuration.html#codec[codec] option for encoding instead.
+# For example, rather than using `format => "plain"` use `codec => "s3_plain"`.
```

Comment:
Probably mention that If you are upgrading from v1 to v2 to use that codec if you want the same format.